arcee/cli/commands/retriever.py

```diff
@@ -0,0 +1,134 @@
+from pathlib import Path
```

Comment:
This is probably abandonable entirely... retriever want to bring it back here eventually - maybe just keep download retriever weights?

Reply:
Yeah I moved anything old and/or DALM related into this retriever command section for better organization, and because I did not know what else to do with em.

Happy to remove all methods besides the ones you tagged here.